src/server/graphql/rootSchema.js

```diff
@@ -12,13 +12,15 @@ import query from './rootQuery';
 import subscription from './rootSubscription';
 import NotifyKickedOut from 'server/graphql/types/NotifyKickedOut';
 import NotifyProjectInvolves from 'server/graphql/types/NotifyProjectInvolves';
+import NotifyFacilitatorDisconnected from 'server/graphql/types/NotifyFacilitatorDisconnected';
 
 export default new GraphQLSchema({
   query,
   mutation,
   subscription,
   types: [
```

Comment:
oh that's such a good question.
If you look at Notification.js, you'll see something that looks very similar to your flow type: https://github.com/ParabolInc/action/blob/data-constructors-and-accessors/src/universal/data/notification.js

To compose the schema, graphql traverses all the nodes & picks up types along the way.
sometimes, a few leaves don't get picked up. this is really common for Interface & Union types. For example, the `NotifyDenial` isn't used by anything except `Notification` when it tries to figure out what kind of notification the thing is. 
This is because in GraphQL land, we say "NotifyDenial uses the Notification interface" but we _don't_ say "Notification implements NotifyDenail, NotifyKickedOut, etc". So, for those leaf type that get left behind, we manually include them here.